File: phpPOO/POOAula10/Aluno.php
<?php
require_once './Pessoa.php';
class Aluno extends Pessoa {

    private $matr;
    private $curso;

    public function cancelarMatr() {
        print "<p>Cancelando Mátricula</p>";
    }
    public function getMatr() {
        return $this->matr;
    }

    public function getCurso() {
        return $this->curso;
    }

    public function setMatr($matr): void {
        $this->matr = $matr;
    }

    public function setCurso($curso): void {
        $this->curso = $curso;
    }


}

File: javaPOO/POOAula10/src/pooaula10/Aluno.java

package pooaula10;

public class Aluno extends Pessoa {
    private int mat;
    private String curso;
    
    public void canelarMatr(){
        System.out.println("Mátricula será cancelada");
    }

    public int getMat() {
        return mat;
    }

    public void setMat(int mat) {
        this.mat = mat;
    }

    public String getCurso() {
        return curso;
    }

    public void setCurso(String curso) {
        this.curso = curso;
    }
    
}
